rules.d/34.repo-slackware.yaml

```diff
@@ -0,0 +1,7 @@
+- { name: mozilla-firefox, setname: firefox, family: slackware }
```

Comment:
Thanks,  I added the same for mozilla-thunderbird and this is now a separate rules file 17.renames-mozilla.yaml